accessibility.html

```diff
@@ -77,6 +78,33 @@
       </div>
     </div>
 
+    <footer class="footer">
+      <div class="columns">
+        <div class="column">
+          <p class="has-text-centered"><a href="privacy-policy.html">Privacy Policy</a></p>
+        </div>
+        <div class="column">
+          <p class="has-text-centered social-links">
+            <a target="_blank" rel="noopener" href="https://www.facebook.com/">
```

Comment:
Just a small thing, but the indentation in these elements seems to be a bit off.  Notice how the indentation (shown below in the code block), changes the how we can have human readable code. This could be 100% down to your formatter settings but just something to be wary of.
```
<a target="_blank" rel="noopener" href="https://www.instagram.com/">
        <i class="fab fa-instagram" aria-hidden="true"></i>
        <span class="sr-only">Instagram</span>
</a>
```

If instances of this could be rectified that would be stellar 🤜

Reply:
Updated